zappa/handler.py

```diff
@@ -336,9 +336,24 @@ def handler(self, event, context):
 
         # This is a direct command invocation.
         elif event.get('command', None):
-
             whole_function = event['command']
+            args = []
+            kwargs = {}
```

Comment:
Sorry, i've been pretty busy and couldn't get back to this.  I do have a ticket this sprint to get back to this, so I can help out tomorrow or this weekend.  That said, here is where I left off:  I wasn't able to get this working, and I don't really remember what the issue was.  I'll take a look and let you know when I can.